docs/analysis_plan.qmd

```diff
@@ -46,30 +46,36 @@ In the SI, we will compare performance of both these models to a baseline auto-A
 ## Implementation
 
 ### Addressing right-truncation
-The hospital admissions data provided are indexed by the data of the positive test -- which is sometimes well before patients end up hospitalised, leading to a right-truncation problem described in [@Wolffram2023].
+The hospital admissions data provided are indexed by the data of the positive test -- which is sometimes well before patients end up hospitalised, leading to a right-truncation problem described in detail in [@Wolffram2023].
 
-We will address this by separately estimating a delay distribution for each region and forecast date using `baselinenowcast`, and modifying the `wwinference` model to account for right-truncated data via incorporating a reporting delay in the observation model.
+We will address this in two ways:
 
+1. By using the vitnaged hospitalization datasets and separately estimating a delay PMF for each region and forecast date using `baselinenowcast`, and modifying the `wwinference` model to account for right-truncated data via incorporating a reporting delay in the observation model. We note that this is the same approach taken in the `EpiNow2` R package, which allows users to pass in a separately estimated delay PMF.
```

Comment:
its one of but not "the" i.e the default assumed workflow is you pass in a delay defined by priors. CFA just chose to use this approach as they don't believe in distributional uncertainty.

Reply:
Will edit!

Reply:
yes I am not suggesting you do anything differently just the wording